faqs/README.md

```diff
@@ -135,18 +135,22 @@ The following prices are exclusive of any local taxes and shipping/handling fees
 | Raspberry Pi 4 Model B 2GB | $35 |
 | Raspberry Pi 4 Model B 4GB | $55 |
 | Raspberry Pi 4 Model B 8GB | $75 |
+| [Raspberry Pi 4 Model B official kits 2/4/8GB](https://www.raspberrypi.org/products/raspberry-pi-4-desktop-kit)  | $100/$120/$140 |
 | Raspberry Pi Zero | $5 |
 | Raspberry Pi Zero W |$10 |
 | Raspberry Pi Zero WH |$15 |
 | Raspberry Pi 400 |$70 |
-| Raspberry Pi 400 (full kit) |$100 |
-
+| [Raspberry Pi 400 offical kit](https://www.raspberrypi.org/products/raspberry-pi-400) |$100 |
 
 ### What do I get when I buy one?
 
-You get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
+Raspberry Pi computers are available separately, or as part of a kit which contain everything you need to get started including a mouse, HDMI cable, SD card, power supply, as well as our official Beginners Guide book. The Raspberry Pi 400 computer contains a case and keyboard built in: other kits come with a separate case and keyboard.
+
+Raspberry Pi also sell all the equipment you need to get you Raspberry Pi up and running separately. We recommend using an official Raspberry Pi Power Supply. For the Pi Zero, 1, 2 and 3 use the [Raspberry Pi 1, 2 and 3 Power Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), for the Pi 4B and Pi 400 use the [Raspberry Pi USB-C Power Supply](https://www.raspberrypi.org/products/type-c-power-supply/).
+
+An official case for the Raspberry Pi is also available separately [from our distributors](https://www.raspberrypi.org/products/). There are also loads of awesome [3D-printable cases](https://www.raspberrypi.org/blog/3d-printed-raspberry-pi-cases/) that community members have designed.
```

Comment:
I wonder if "An official case" should be "Official cases", since there's different official cases for the Zero, A+, 3B and 4B?
("An official case" might imply that there's a single case available for all models of Raspberry Pi? Or perhaps I'm just being too pedantic :wink: )

Reply:
We cannot list everything that third parties do, especially if we now sell our own versions, so inclined to slowly remove third party references.

Reply:
If 3rd-party products aren't being mentioned, then it feels a bit odd to still be mentioning 3D-printed cases? (Kind of implies that there's nothing in-between.) Or maybe that's just me :man_shrugging:

Reply:
I'm happy to keep including community stuff, which is why I left it in. Maybe it could be phrased better.

Reply:
Could well be. A question for less pressured times :)

Reply:
I don't think we should try to be too completist here, or aspire to flawless consistency - not compatible with (a) reality (b) making a reasonable stab at being actually helpful to readers who need it.

Reply:
Having a quick think, and I wonder if getting rid of the "there are also" phrasing would assuage my niggle?
E.g. change "There are also loads of awesome [3D-printable cases](https://www.raspberrypi.org/blog/3d-printed-raspberry-pi-cases/) that community members have designed." to "Community members have designed loads of awesome [3D-printable cases](https://www.raspberrypi.org/blog/3d-printed-raspberry-pi-cases/)." :man_shrugging: